docs/source/contents/4_draft_mapping.rst

```diff
@@ -1777,10 +1777,644 @@ Decision
 
 **Do not migrate!**
 
-
 subject
 =======
 
+None type
+---------
+
+Use Case
+^^^^^^^^
+
+Several elements contain unintentional null values. There are five within Tennessee Documentary History. Additional null
+subjects include vpmoore:133 and adams:76. Most of roth seems to have null mods:subject/mods:name/mods:namePart values.
+It appears we might have inserted some blank nodes using the Islandora form entry. As there is no information, these
+"values" are not used and have no true use case.
+
+Justification
+^^^^^^^^^^^^^
+
+These nodes contain no information.
+
+Xpath
+^^^^^
+
+    :code:`subject/topic[text()='']` OR
+    :code:`subject/geographic[text()='']` OR
+    :code:`subject/name/namePart[text()='']`
```

Comment:
I know that I'm being a bit ridiculous, but this syntax isn't right. Let's change it; all of the following versions are better:
`subject/topic[not(text())]` or
`subject/topic[string() = '']` or
`subject/topic[. = '']`

The second one is probably the best and/or safest.

Reply:
Thanks - I went with the second as suggested.

Reply:
I'm good but in case you're curious from the definition of the string function on  W3C:

> Every node has a string value, even an element with element-only content (which has no typed value). Moreover, casting an atomic value to a string always succeeds. Functions, maps, and arrays have no string value, so these are the only arguments that satisfy the type signature but cause failure.

Reply:
My initial instinct was subject/topic[not(text())], but I could not figure out which would be "safest". I would say from a consistency standpoint we do have [not(text())] elsewhere in our mapping. We should probably address this in the same way throughout just so that the doc is easier to digest.